assets/dev/js/editor/components/ai/components/prompt-action-selection.js

```diff
@@ -0,0 +1,50 @@
+import { MenuItem, Select, FormControl, InputLabel } from '@elementor/ui';
+
+const labelToDashCash = ( str ) => str.toLowerCase().replace( /\s/g, '-' );
+
+const PromptActionSelection = ( props ) => {
+	const actionId = labelToDashCash( props.label );
+
+	return (
+		<FormControl sx={ { width: 138 } }>
+			<InputLabel id={ actionId }>{ props.label }</InputLabel>
+
+			<Select
+				labelId={ actionId }
+				id={ actionId }
+				value=""
+				color="secondary"
+				onChange={ props.onChange }
+				size="small"
+				label={ props.label }
+				MenuProps={ {
+					PaperProps: {
+						sx: {
+							width: 138,
```

Comment:
can't it somehow inherit this value from the parent?

Reply:
Can't think of a way..